src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -120,12 +120,27 @@ def get_pool_list_with_stats(cls, application=None):
 
     @classmethod
     def get_pool_name_from_id(cls, pool_id):
+        # type: (int) -> Union[str, None]
+        pool = cls.get_pool_by_attribute('pool', pool_id)
```

Comment:
There's a librados call exactly for this: `self.rados.pool_reverse_lookup(pool_id)`.

Reply:
I will wait for more input through your other comment before addressing this one.

Reply:
Hi I tried your suggestion but it failed with `module 'rados' has no attribute 'pool_reverse_lookup'`. So I kept it as it is.